.github/workflows/publish.yml

```diff
@@ -33,7 +26,7 @@ jobs:
       - name: Publish NuGet packages
         run: |
           mkdir build
-          dotnet pack -c Release -o build
+          dotnet pack -c Release -o build /p:Version=${{ github.event.release.tag_name }}
```

Comment:
this doesn't work for two reasons.
1. `-o` is unsupported on solution level, that's what we use artifacts output for. no more `build` folders.
2. `github.event.release.tag_name` will usually contain a preceding `v`; and we might want to version different packages differently. imo the best way to handle this is to just not touch it at all and leave it up to the csproj files

Reply:
1. `-o` is very much supported on a solution level
2. ok

Reply:
1. mb, i forgot that sdk 7.0.201 reenabled it; but we should still use artifacts output for the reasons listed on discord